terraform/aws/variables.tf

```diff
@@ -59,3 +59,10 @@ variable machines_wc {
     })
   }))
 }
+
+variable extra_tags {
+  default = {
+  }
```

Comment:
@Xartos See here. `extra_tags` is initialized by default to an empty map.

Reply:
Right, yea it's just me that's blind...